L1Trigger/TrackerTFP/interface/HoughTransform.h

```diff
@@ -0,0 +1,59 @@
+#ifndef L1Trigger_TrackerTFP_HoughTransform_h
+#define L1Trigger_TrackerTFP_HoughTransform_h
+
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+
```

Comment:
Add #include for TTDTC.h, to make it clear where this class is defined.

Reply:
done